src/commands/orgs/list.ts

```diff
@@ -4,40 +4,17 @@ import { MobileCenterClient, models, clientRequest } from "../../util/apis";
 
 const debug = require("debug")("mobile-center-cli:commands:orgs:list");
 import { inspect } from "util";
+import { getOrgsNamesList } from "./lib/org-users-helper";
 
 @help("Lists organizations in which current user is collaborator")
 export default class OrgListCommand extends Command {
   async run(client: MobileCenterClient): Promise<CommandResult> {
     // every user is a collaborator of it's own group and of zero or more external groups
-    const orgs = await out.progress("Loading list of organizations...", this.getOrgsNamesList(client));
+    const orgs = await out.progress("Loading list of organizations...", getOrgsNamesList(client, debug));
```

Comment:
Debug is used per file - if you pass it around you potentially screw up the logging since we don't know what file the log is coming from anymore. Just use the debug const locally in the function you're calling.